modelzoo/DLRM/README.md

```diff
@@ -39,60 +57,90 @@ The triangles represent mlp network. The inputs consists of dense features and s
-    - `--input_layer_partitioner`: Slice size of input layer partitioner(units MB).
-    - `--dense_layer_partitioner`: Slice size of dense layer partitioner(units kB).
-    - `--protocol`: Set the protocol("grpc", "grpc++", "star_server") used when starting server in distributed training. Default to grpc. 
+    - DeepRec Features:
+      - `export START_STATISTIC_STEP` and `export STOP_STATISTIC_STEP`: Set ENV to configure CPU memory optimization. This is already set to 100 & 110 in the code by default.
+      - `--bf16`: Enable DeepRec BF16 feature in DeepRec. Use FP32 by default.
+      - `--emb_fusion`: Whether to enable embedding fusion, Default to True.
+      - `--op_fusion`: Whether to enable Auto graph fusion feature. Default to True.
+      - `--optimizer`: Choose the optimizer for deep model from ['adam', 'adamasync', 'adagraddecay', 'adagrad', 'gradientdescent']. Use adamasync by default.
+      - `--smartstaged`: Whether to enable smart staged feature of DeepRec, Default to True.
+      - `--micro_batch`: Set num for Auto Mirco Batch. Default 0 to close.(Not really enabled)
+      - `--ev`: Whether to enable DeepRec EmbeddingVariable. Default to False.
+      - `--adaptive_emb`: Whether to enable Adaptive Embedding. Default to False.
+      - `--ev_elimination`: Set Feature Elimination of EmbeddingVariable Feature. Options [None, 'l2', 'gstep'], default to None.
+      - `--ev_filter`: Set Feature Filter of EmbeddingVariable Feature. Options [None, 'counter', 'cbf'], default to None.
+      - `--dynamic_ev`: Whether to enable Dynamic-dimension Embedding Variable. Default to False.(Not really enabled)
+      - `--incremental_ckpt`: Set time of save Incremental Checkpoint. Default 0 to close.
+      - `--workqueue`: Whether to enable Work Queue. Default to False.
+      - `--protocol`: Set the protocol ['grpc', 'grpc++', 'star_server'] used when starting server in distributed training. Default to grpc. 
+    - Basic Settings:
+      - `--data_location`: Full path of train & eval data, default to `./data`.
+      - `--steps`: Set the number of steps on train dataset. Default will be set to 10 epoch.
+      - `--no_eval`: Do not evaluate trained model by eval dataset.
+      - `--batch_size`: Batch size to train. Default to 512.
+      - `--output_dir`: Full path to output directory for logs and saved model, default to `./result`.
+      - `--checkpoint`: Full path to checkpoints input/output directory, default to `$(OUTPUT_DIR)/model_$(MODEL_NAME)_$(TIMESTAMPS)`
+      - `--save_steps`: Set the number of steps on saving checkpoints, zero to close. Default will be set to 0.
+      - `--seed`: Set the random seed for tensorflow.
+      - `--timeline`: Save steps of profile hooks to record timeline, zero to close, defualt to 0.
+      - `--keep_checkpoint_max`: Maximum number of recent checkpoint to keep. Default to 1.
+      - `--learning_rate`: Learning rate for network. Default to 0.1.
```

Comment:
add `--interaction_op`: Choose interaction op before top MLP layer("dot", "cat"). Default to "cat".